.github/workflows/release-canary.yml

```
@@ -0,0 +1,53 @@
+          submodules: recursive
+
+      - name: 'Setup'
+        uses: ./.github/actions/setup
+
+      - name: Set version
+        run: npm --no-git-tag-version version ${{ env.PACKAGE_VERSION }}-canary.$(date +'%Y%m%d') -w packages/wallet-sdk
+
+      # Change to the wallet-sdk directory
+      - name: Change to wallet-sdk directory
+        run: cd packages/wallet-sdk
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)